cmd/debug/debug.go

```diff
@@ -0,0 +1,215 @@
+package debug
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	dbCmd "github.com/trento-project/trento/cmd/db"
+	"github.com/trento-project/trento/internal"
+	"github.com/trento-project/trento/internal/db"
+	"github.com/trento-project/trento/web"
+	"github.com/trento-project/trento/web/datapipeline"
+	"github.com/trento-project/trento/web/entities"
+	"gorm.io/gorm"
+)
+
+func NewDebugCmd() *cobra.Command {
+	debugCmd := &cobra.Command{
+		Use:   "debug",
+		Short: "Maintanance commands, USE WITH CAUTION.",
+		PersistentPreRunE: func(debugCmd *cobra.Command, _ []string) error {
+			debugCmd.Flags().VisitAll(func(f *pflag.Flag) {
+				viper.BindPFlag(f.Name, f)
+			})
+			debugCmd.PersistentFlags().VisitAll(func(f *pflag.Flag) {
+				viper.BindPFlag(f.Name, f)
+			})
+			return internal.InitConfig("web")
```

Comment:
Is this `web` a leftover, or something we need to have there?

Reply:
good call, usual leftover thanks!